src/vs/workbench/contrib/terminal/test/browser/links/terminalWordLinkProvider.test.ts

```diff
@@ -282,13 +279,23 @@ suite('Workbench - TerminalWordLinkProvider', () => {
 		await assertLink('file:///C:/users/test/file.txt ', [{ range: [[1, 1], [30, 1]], text: 'file:///C:/users/test/file.txt' }]);
 		await assertLink('file:///C:/users/test/file.txt:1:10 ', [{ range: [[1, 1], [35, 1]], text: 'file:///C:/users/test/file.txt:1:10' }]);
 	});
-	test.skip('should add cwd to link', async () => {
-		xterm.commandTracker.setCwd('/Users/home/folder');
-		await assertLink('file.txt ', [{ range: [[1, 1], [8, 1]], text: 'file.txt', linkActivationResult: { link: '/Users/home/folder/file.txt', source: 'editor' } }], true);
+	test('should apply the cwd to the link only when the file exists and cwdDetection is enabled', async () => {
+		if (!isWindows) {
+			const { text, cwd, filePath } = generateLinkArgs(['Users', 'home', 'folder']);
+			await assertLink(text, [{ range: [[1, 1], [8, 1]], text, linkActivationResult: { link: filePrefix + filePath, source: 'editor' } }], true, cwd, [filePath]);
+			await assertLink(text, [{ range: [[1, 1], [8, 1]], text, linkActivationResult: { link: text, source: 'quickpick' } }], true, cwd, []);
+		}
+	});
+	test('should not add the cwd to the link when cwdDetection is not enabled', async () => {
+		const { text, cwd, filePath } = generateLinkArgs(['Users', 'home', 'folder']);
+		await assertLink(text, [{ range: [[1, 1], [8, 1]], text, linkActivationResult: undefined }], false, cwd, [filePath]);
+		await assertLink(text, [{ range: [[1, 1], [8, 1]], text, linkActivationResult: undefined }], false, cwd, []);
 	});
```

Comment:
these are failing on windows

Reply:
it looks like `isAbsolute` calls into this, which won't work 

https://github.com/microsoft/vscode/blob/343cf36472de32ac584a38c735905fe7d8030953/src/vs/base/common/path.ts#L213-L219

Reply:
so only running these for non-windows